tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py

```diff
@@ -16,6 +16,11 @@
 batch_request = data_asset.build_batch_request()
 # </snippet>
 
+# <snippet name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py create_expectation_suite">
+context.create_expectation_suite("my_expectation_suite")
+assert "my_expectation_suite" in context.list_expectation_suite_names()
```

Comment:
I would move this assert statement outside of the snippet.  We don't need to display it in the docs: it's just there to ensure that the code works as intended when we are testing the snippet.

Reply:
@Rachel-Reverie I've implemented the edit as requested.